app/src/app/constants/layers.ts

```diff
@@ -13,14 +12,59 @@ export const DEFAULT_PAINT_STYLE: ExpressionSpecification = [
 ];
 
 export const ZONE_ASSIGNMENT_STYLE: ExpressionSpecification = [
-  // based on zone feature state, set fill color
   "case",
-  ["==", ["feature-state", "zone"], 1],
-  "#0099cd",
-  ["==", ["feature-state", "zone"], 2],
-  "#ffca5d",
-  ["==", ["feature-state", "zone"], 3],
-  "#00cd99",
+  ["==", ["feature-state", "zone"], "#dd4425"],
```

Comment:
O/question: curious if we can get the hex code from the radix color objects in order to directly tie the palettes from `app/src/app/constants/colors.ts` to this.

Reply:
set dynamically now, much less brittle